### PR TITLE
Add TLS documentation

### PR DESCRIPTION
**Pull request by vincentkam**

### CODE REVIEW

File: Docs/reference/content/reference/driver/ssl.md [hunk 1]
```diff
@@ -40,4 +40,45 @@ var settings = new MongoClientSettings
...
 ```
 
-{{% note class="important" %}}It is imperative that when loading a certificate with a password, the [PrivateKey]({{< msdnref "system.security.cryptography.x509certificates.x509certificate2.privatekey" >}}) property not be null. If the property is null, it means that your certificate does not contain the private key and will not be passed to the server.{{% /note %}}
\ No newline at end of file
+{{% note class="important" %}}It is imperative that when loading a certificate with a password, the [PrivateKey]({{< msdnref "system.security.cryptography.x509certificates.x509certificate2.privatekey" >}}) property not be null. If the property is null, it means that your certificate does not contain the private key and will not be passed to the server.{{% /note %}}
+
+## TLS support
+### Overview
+
+| OS      | .NET Version          | TLS1.1 | TLS1.2 | SNI | CRLs sans OCSP |
```
Comment (jyemin) [inline Docs/reference/content/reference/driver/ssl.md]: "sans"?  

If by that you mean "without", please replace with "without".  :)

**Review by jyemin — CHANGES_REQUESTED** [review]
Have you looked at a rendered view of the table?

File: Docs/reference/content/reference/driver/ssl.md (hunk 1 above)
Comment (vincentkam) [reply Docs/reference/content/reference/driver/ssl.md]: Fixed :)

**Review by jyemin — APPROVED** [review]

**Comment by jyemin** [comment]
Assuming the rendered view of the table looks ok, LGTM.

**Comment by vincentkam** [comment]
Rendered view of the table looks good with Hugo 0.13!

**Pull request merged**